README.md

```diff
@@ -138,6 +139,7 @@ PRs accepted.
 Small note: If editing the README, please conform to the
 [standard-readme](https://github.com/RichardLitt/standard-readme) specification.
 
+
```

Comment:
@Ultra-Tech-code I believe this violates the rules of our prettier config.  Sorry for all the run around, but can you run `npm run prettier:fix` again?

Reply:
fix